api/response/schema.py

```diff
@@ -17,6 +18,30 @@ class Meta:
         model = ResponseModel
 
 
+class ResponseDetail(graphene.ObjectType):
+    response_id = graphene.Int()
+    suggestion = graphene.String()
+    missing_items = graphene.List(graphene.String)
+    created_date = graphene.DateTime()
+    rating = graphene.Int()
+    resolved = graphene.Boolean()
+
+
+class RoomResponses(graphene.ObjectType):
+    room_id = graphene.Int()
+    room_name = graphene.String()
+    total_responses = graphene.Int()
+    response = graphene.List(ResponseDetail)
+
+
+class PaginatedResponse(graphene.ObjectType):
+    pages = graphene.Int()
+    query_total = graphene.Int()
+    has_next = graphene.Boolean()
```

Comment:
What do you think about adding current_page field?

Reply:
I think that makes sense. My next push will cover that.